cmd/release/internal/release.go

```diff
@@ -92,14 +94,15 @@ func newRelease(inputBranch, inputEnvironment string, overrideNumber int) (*Rele
 	release.BranchEKSNumber = fmt.Sprintf("%s-%s", branchEKS, release.number)
 	release.BranchEKSPreviousNumber = fmt.Sprintf("%s-%s", branchEKS, release.previousNumber)
 	release.BranchWithDot = strings.Replace(release.branch, "-", ".", 1)
+	release.BranchWithDotNumber = fmt.Sprintf("%s-%s", release.BranchWithDot, release.number)
 	release.EKSBranchNumber = fmt.Sprintf("eks-%s-%s", release.branch, release.number)
 	release.EKSBranchPreviousNumber = fmt.Sprintf("eks-%s-%s", release.branch, release.previousNumber)
 	release.K8sBranchEKS = "kubernetes-" + branchEKS
 	release.K8sBranchEKSNumber = fmt.Sprintf("%s-%s", release.K8sBranchEKS, release.number)
 	release.K8sBranchEKSPreviousNumber = fmt.Sprintf("%s-%s", release.K8sBranchEKS, release.previousNumber)
 	release.VBranchEKSNumber = "v" + release.BranchEKSNumber
 	release.VBranchEKSPreviousNumber = "v" + release.BranchEKSPreviousNumber
-	release.VBranchWithDotNumber = fmt.Sprintf("v%s-%s", release.BranchWithDot, release.number)
+	release.VBranchWithDotNumber = "v" + release.BranchWithDotNumber
```

Comment:
So we don't need release.number here anymore?

Reply:
`release.BranchWithDotNumber` includes number